### PR TITLE
Adds possibility to order follow up forms by their name.

### PR DESCRIPTION
**Pull request by sztamas**
The follow up forms are currently ordered by creation time.

This feature allows the registry owners to change the ordering
of the links to the follow up forms by using:

Admin -> Registry Context Form Groups

The ordering can be set to "Creation Time", which is the current
behaviour, or to "Name" which will order the links by their name, which
for FH is the Date of Assessment.

### CODE REVIEW

File: rdrf/rdrf/exporter.py [hunk 1]
```diff
@@ -496,6 +496,7 @@ def _get_context_form_groups(self):
             cfg_dict["forms"] = []
             for form in cfg.forms:
                 cfg_dict["forms"].append(form.name)
+            cfg_dict["ordering"] = cfg.ordering
```
Comment (id2359) [inline rdrf/rdrf/exporter.py]: did you make a change to the importer also?

File: rdrf/rdrf/models.py [hunk 2]
```diff
@@ -1949,13 +1960,16 @@ class ContextFormGroup(models.Model):
                       ("N", "Automatic - Number"),
                       ("M", "Manual - Free Text"),
                       ("C", "CDE - Nominate CDE to use")]
+    ORDERING_TYPES = [("C", "Creation Time"),
```
Comment (id2359) [inline rdrf/rdrf/models.py]: cool

File: rdrf/rdrf/models.py [hunk 3]
```diff
@@ -2002,6 +2024,21 @@ def get_default_name(self, patient_model):
         else:
             return "Modules"
 
+    def get_value_from_cde(self, patient_model, context_model):
+        form_name, section_code, cde_code = self.naming_cde_to_use.split("/")
+        try:
+            cde_value = patient_model.get_form_value(self.registry.code,
```
Comment (id2359) [inline rdrf/rdrf/models.py]: Is it possible to specify a cde in a multisection in the field ?

File: rdrf/rdrf/exporter.py (hunk 1 above)
Comment (sztamas) [inline rdrf/rdrf/exporter.py]: hm, I was sure that I did and that I tested it, but lost the change somehow.
Thanks for picking it up :)

File: rdrf/rdrf/models.py (hunk 3 above)
Comment (sztamas) [inline rdrf/rdrf/models.py]: well, not sure. I did what the `get_name_from_cde` below was doing.
Should these be changed?

Comment (id2359) [inline rdrf/rdrf/models.py]: yeah I don't think I added any validation at all for that - so leave it as is

**Comment by id2359** [comment]
approved from me

**Pull request merged**